src/saltext/vmware/modules/vmc_sddc.py

```diff
@@ -225,14 +225,35 @@ def create(
 
     deployment_type: String
         (Optional) Denotes if request is for a SingleAZ or a MultiAZ SDDC. Default is SingleAZ.
-        Possible values are: SingleAZ , MultiAZ
+        Possible values are: SingleAZ, MultiAZ
 
     host_instance_type: String
-        (Optional) The instance type for the esx hosts in the primary cluster of the SDDC.
+        (Optional) The instance type for the ESX hosts in the primary cluster of the SDDC.
```

Comment:
`ESXi`?

Reply:
Here it has been referred to ESX only

Reply:
That must have been changed - there's no reference to ESX at all on https://developer.vmware.com/apis/vmc/v1.1/data-structures/SddcConfig/

Based on information https://www.nakivo.com/blog/vmware-esx-vs-esxi-key-differences-overview/ I'm not sure that ESX is the correct term anymore :thinking:

Reply:
yeah this upstream document has changed since we have implemented sal module

According to new doc there is no explanination of **host_instance_type** 
just possible values are there : i3.metal , r5.metal , i3en.metal

So i have also updated removing **ESX** term itself as we are not  very sure what it should referring here
 The instance type for the hosts in the primary cluster of the SDDC.
 Possible values are: i3.metal, r5.metal, i3en.metal